MainRepository/Program.cs

```diff
@@ -1,7 +1,21 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿/// <summary>Implementation of Anagram</summary>
+class Anagram {
+   public static void Main () {
+      Anagram a = new ();
+      a.FindAnagrams ();
+   }
+
+   /// <summary>This finds all anagrams and sorts them based on anagram count</summary>
+   public void FindAnagrams () {

```

Comment:
As said earlier, this method not required instead this code could be inside Main itself